src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import core.basesyntax.interfaces.Calculatable;
+import core.basesyntax.interfaces.Drawable;
+
+public class Circle extends Figure implements Calculatable, Drawable {
```

Comment:
all our figures should implement interfaces `Drawable` and `Calculatable` - how can we optimise the code so that we won’t have to write `implements Drawable, Calculatable` for Circle and other child figures?